### PR TITLE
Strided slice

### PR DESCRIPTION
**Pull request by aranXmos**

### CODE REVIEW

File: cmakefiles/xtflm.cmake [hunk 1]
```diff
@@ -191,3 +192,5 @@ list(APPEND ALL_INCLUDES  "${CMAKE_CURRENT_SOURCE_DIR}/../lib_tflite_micro/submo
 list(APPEND ALL_INCLUDES  "${CMAKE_CURRENT_SOURCE_DIR}/../../lib_nn/")
 list(APPEND ALL_INCLUDES  "${CMAKE_CURRENT_SOURCE_DIR}/../../lib_nn/lib_nn/api")
 list(APPEND ALL_INCLUDES  "${XMOS_TOOL_PATH}/target/include/")
+
```
Comment (panickal-xmos) [inline cmakefiles/xtflm.cmake:195]: Do we need this change? Did we modify this?

File: lib_tflite_micro/src/tflite-xcore-kernels/xcore_strided_slice.cc [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+#include "tensorflow/lite/kernels/kernel_util.h"
+#include "tensorflow/lite/micro/kernels/kernel_util.h"
+#include "xcore_custom_options.h"
+#include "xcore_interpreter.h"
+#include "xcore_dispatcher.h"
+#include "xcore_utils.h"
+extern "C" {
+#include "nn_operator.h"
+}
+#include <cstdio>
```
Comment (panickal-xmos) [inline lib_tflite_micro/src/tflite-xcore-kernels/xcore_strided_slice.cc:16]: Please remove the headers we are not using. I think many can be removed.

File: lib_tflite_micro/src/tflite-xcore-kernels/xcore_strided_slice.cc [hunk 3]
```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2022, XMOS Ltd, All rights reserved
```
Comment (panickal-xmos) [inline lib_tflite_micro/src/tflite-xcore-kernels/xcore_strided_slice.cc:1]: Could you please run clang-format on this file? Please let me know if you would like help with it.

Comment (aranXmos) [reply lib_tflite_micro/src/tflite-xcore-kernels/xcore_strided_slice.cc:1]: Done

File: cmakefiles/xtflm.cmake (hunk 1 above)
Comment (aranXmos) [reply cmakefiles/xtflm.cmake:195]: Not required, removed

File: lib_tflite_micro/src/tflite-xcore-kernels/xcore_strided_slice.cc (hunk 2 above)
Comment (aranXmos) [reply lib_tflite_micro/src/tflite-xcore-kernels/xcore_strided_slice.cc]: Removed

**Comment by panickal-xmos** [comment]
Thank you @aranXmos!

**Review by panickal-xmos — APPROVED** [review]
+1

**Pull request merged**